doc/source/cluster/cluster_under_construction/key-concepts.rst

```diff
@@ -1,130 +1,104 @@
+    
+    *A Ray cluster with two worker nodes. Each node runs Ray helper processes to
+    facilitate distributed scheduling and memory management. The head node runs
+    additional control processes, which are highlighted.*
 
-.. _cluster-head-node-under-construction:
-
-Head node
-~~~~~~~~~
+The number of worker nodes in a cluster may change with application demand, according
+to your Ray cluster configuration. This is known as *autoscaling*. The head node runs
+the :ref:`autoscaler<cluster-autoscaler-under-construction>`.
 
-The head node is the first node started by the
-:ref:`Ray cluster launcher<cluster-launcher>` when trying to launch a Ray
-cluster. Among other things, the head node holds the :ref:`Global Control Store
-(GCS)<memory>` and runs the :ref:`autoscaler<cluster-autoscaler>`. Once the head
-node is started, it will be responsible for launching any additional
-:ref:`worker nodes<cluster-worker-node>`. The head node itself will also execute
-tasks and actors to utilize its capacity.
+.. note::
+    Ray nodes are implemented as pods when :ref:`running on Kubernetes<kuberay-index>`.
 
-.. _cluster-worker-node-under-construction:
+Users can submit jobs for execution on the Ray cluster, or can interactively use the
+cluster by connecting to the head node and running `ray.init`. See
+:ref:`Clients and Jobs<cluster-clients-and-jobs-under-construction>` for more information.
 
-Worker node
-~~~~~~~~~~~
+.. _cluster-worker-nodes-under-construction:
 
-A worker node is any node in the Ray cluster that is not functioning as head node.
-Therefore, worker nodes are simply responsible for executing tasks and actors.
-When a worker node is launched, it will be given the address of the head node to
-form a cluster.
+Worker nodes
+~~~~~~~~~~~~
+**Worker nodes** are nodes which run Ray and are connected to the head node. Ray may schedule
+tasks and actors onto worker nodes. Each worker node runs helper processes which
+implement distributed scheduling and :ref:`memory management<memory>`.
 
-.. _cluster-launcher-under-construction:
+.. _cluster-head-node-under-construction:
 
-Cluster launcher
-----------------
+Head node
+~~~~~~~~~
+Every Ray cluster has one node which is designated as the **head node** of the cluster.
+The head node runs singleton processes responsible for cluster management such as the
+:ref:`autoscaler<cluster-autoscaler-under-construction>` and the Ray driver processes
+:ref:`which run Ray jobs<cluster-clients-and-jobs-under-construction>`. Ray may schedule
+tasks and actors on the head node just like any other worker node, unless configured otherwise.
 
-The cluster launcher is a process responsible for bootstrapping the Ray cluster
-by launching the :ref:`head node<cluster-head-node>`. For more information on how
-to use the cluster launcher, refer to
-:ref:`cluster launcher CLI commands documentation<cluster-commands>` and the
-corresponding :ref:`documentation for the configuration file<cluster-config>`.
+.. note::
```

Comment:
Do we need this note, @DmitriGekhtman? It seems pretty off-topic / too much detail for a first-time user.

Reply:
Alternatively, we could move it as an inline note to the "Autoscaler" subsection (suggestion below).

Reply:
I committed your suggestion and removed this note!